app/src/main/java/app/grapheneos/apps/ui/MainActivity.kt

```diff
@@ -35,6 +36,16 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         val views = MainActivityBinding.inflate(layoutInflater)
         this.views = views
+        //get the current intent
+        val intent = intent
```

Comment:
?

Reply:
"val intent = intent" is needed for intent.getBooleanExtra()